code/src/components/MusicInfo.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+import { Albums } from './Albums'
+
+/* Creating the main album wrapper and mapping through it, to be accessed in Albums.js */
+const MusicInfo = (props) => {
+  return (
+    <section className="albums-wrapper">
+    {props.musicList.albums.items.map((item) => {
+         return <Albums key={item.id} albumTitle={item.name} 
+         image={item.images[1].url} artists={item.artists[0].name} 
```

Comment:
Nice, it's easy to miss out on but some albums actually have more than one artist (e.g. artists[0], artists[1] etc.). One idea is that you could make another map for the artists array, and perhaps separate the artist info into another component to import to make it easier to follow the mapping.

Reply:
! Totally did not see this part - thank you so much for pointing this out! On it right away!